MatterControl.Printing/PrinterCommunication/PrinterConnection.cs

```diff
@@ -941,6 +926,18 @@ public void ReleaseAndReportFailedConnection(ConnectionFailure reason)
 			OnConnectionFailed(reason);
 		}
 
+		public void LogInfo(string message)
+		{
+			// TODO: Add messaging
```

Comment:
Create issue

Reply:
Issue Created - MatterHackers/MCCentral#5766
Replace PrinterConnection MatterControl error messages with error and specific error codes